dramatiq/registry.py

```diff
@@ -0,0 +1,65 @@
+from .actor import Actor, _queue_name_re
+
+
+class Registry:
```

Comment:
Great to see this! I nearly have the same class in my projects.

I have implemented this method as well:

```
    def enqueue(self, message, *, delay=None):
        raise RuntimeError(
            "ActorCollector has not transferred its actors to an actual broker. "
            "Ensure transfer_actors() is called."
        )
```

It ensures that the user is told what is going on when trying to call send() on an actor before doing the bind.